dashboard/app/models/ability.rb

```diff
@@ -198,6 +198,14 @@ def initialize(user)
         user.persisted? || !script.login_required?
       end
     end
+    can :student_lesson_plan, Lesson do |lesson|
+      script = lesson.script
+      if script.pilot?
+        script.has_pilot_access?(user)
+      else
+        user.persisted? || !script.login_required?
```

Comment:
I think we will want this to be just `else true end` because we don't need to check that a user in logged in to view this content